src/AI/ModularAI.java

```diff
@@ -68,7 +68,7 @@ public GameAction getNextAction(GameMap gameMap)
     }
 
     GameAction nextAction = null;
-    PriorityQueue<Unit> unitQueue = new PriorityQueue<Unit>(13, unitOrderSetter);
+    PriorityQueue<Unit> unitQueue = new PriorityQueue<Unit>(eligibleUnits.size(), unitOrderSetter);
```

Comment:
This works great after the first turn, but you should really enforce a minimum of 1 (because 0 causes a stack trace).

Reply:
Huh. That's dumb. I figured it was too innocuous to break anything, _sigh_.